src/commands/moderators/report.ts

```diff
@@ -18,33 +18,41 @@ export default {
     const member = message?.member
     if (!member) return
 
-    const increment = async (key: string) => {
-      const count = (await keyv.get(key)) ?? 0
-      await keyv.set(key, count + 1)
-      return count + 1
-    }
+    const messageId = BigInt(message.id)
 
     // Count the number of times a message has been reported
-    const messageReportCount = await increment(
-      `reportCount:message:${message.id}`,
-    )
+    const messageReportCount = await prisma.messageReportCount.upsert({
+      where: { messageId },
+      update: { count: { increment: 1 } },
+      create: { messageId, count: 1 },
+    })
+
+    const reporteeMemberId = BigInt(member.id)
 
     // TODO: Count the number of times a user has been reported
```

Comment:
If it is done, please remove "TODO :" and left only useful comment :+1:

Reply:
Done